### PR TITLE
sv_timestamp_analysis: check for negative latencies

### PR DESCRIPTION
**Pull request by Paullgk**
When PTP is not accurate enough, negative latencies can occurred, and the latency test should be considered as failed. However, there is no such test, and thus, CI generate incorrect latency tests results. To fix this, add a test to check for negative latencies.

### CODE REVIEW

**Review by eroussy — COMMENTED** [review]
Do you think it is really relevant to add a line in the test report to look for negative latencies ?
IMO the important thing is that the script fails if it detect any negative latency

**Comment by Paullgk** [comment]
> Do you think it is really relevant to add a line in the test report to look for negative latencies ? IMO the important thing is that the script fails if it detect any negative latency

yes, because IMO it help to diagnostic what went wrong; 
In case on negative latencies, the report is not very clear to understand because no where it its mentioned that it failed because of the neg latencies

File: sv_timestamp_analysis.py [hunk 1]
```diff
@@ -199,6 +199,16 @@ def generate_adoc(pub, hyp, sub, streams, hyp_name, sub_name, output, max_latenc
             """
         )
 
+        pass_line_min_latency = textwrap.dedent(
+            """
+            [cols="3,1",frame=all, grid=all]
+            |===
+            |Min latency > 0 us
```
Comment (eroussy) [inline sv_timestamp_analysis.py:206]: "< 0" not "> 0"

**Review by eroussy — CHANGES_REQUESTED** [review]

**Comment by eroussy** [comment]
> > Do you think it is really relevant to add a line in the test report to look for negative latencies ? IMO the important thing is that the script fails if it detect any negative latency
> 
> yes, because IMO it help to diagnostic what went wrong; In case on negative latencies, the report is not very clear to understand because no where it its mentioned that it failed because of the neg latencies

Agreed
But are you sure that the script returns error in that case ? I only see adoc modification in the PR, not return code or exit

File: sv_timestamp_analysis.py (hunk 1 above)
Comment (Paullgk) [reply sv_timestamp_analysis.py:206]: Don't agree:
"Min latency > 0 us" means: PASS if the min latency is strictly positive and FAILED in the opposite case

**Comment by Paullgk** [comment]
> > > Do you think it is really relevant to add a line in the test report to look for negative latencies ? IMO the important thing is that the script fails if it detect any negative latency
> > 
> > 
> > yes, because IMO it help to diagnostic what went wrong; In case on negative latencies, the report is not very clear to understand because no where it its mentioned that it failed because of the neg latencies
> 
> Agreed But are you sure that the script returns error in that case ? I only see adoc modification in the PR, not return code or exit

Why it would return an error? IMO this is not a "script" error but rather an incorrect latency test, which should be triggered as failed by the report and thus the CI

File: sv_timestamp_analysis.py (hunk 1 above)
Comment (eroussy) [reply sv_timestamp_analysis.py:206]: understood

**Review by eroussy — APPROVED** [review]
